tests/tests_e3/electrolyt/plan/main_test.py

```diff
@@ -161,3 +161,24 @@ def test_verify_entry_point():
         context.execute(my_plan, 'foo', verify=True)
 
     assert 'foo is not an entry point' in str(plan_err)
+
+
+def test_plan_disable_lines():
+    """Check that lines in enabled=False blocks are disabled."""
+    context = _get_new_plancontext('myserver')
+
+    myplan = _get_plan(
+        data={},
+        content=[u'def myserver():\n',
+                 u'    build("foo")\n'
+                 u'    with defaults(enabled=False):\n'
+                 u'        build("bar1")\n'
+                 # It is not possible to "revert" the enabled=False status
+                 u'        with defaults(enabled=True):\n'
+                 u'            build("bar2")\n'
+                 u'    build("bar3")\n'])
+
+    actions = context.execute(myplan, 'myserver', verify=False)
```

Comment:
So, IIUC, calling execute with verify=True, we should be getting 'foo', 'bar1', 'bar2', and 'bar3', right? Should we add a test for that?

Reply:
No, you have this result when you instantiate PlanContext with ignore_disabled=False, I've added a test.